packages/init/templates/package.json.hbs

```diff
@@ -15,7 +15,7 @@
     {{#if babel}}
       "babel": "babel src --ignore **/__tests__ --delete-dir-on-start{{#if typescript}} -x .js,.ts,.tsx{{/if}}",
       {{#if typescript}}
-      "build:types": "tsc --outDir lib{{#if esm}} && tsc --outDir es{{/if}}",
+      "build:types": "tsc --outDir lib --emitDeclarationOnly{{#if esm}} && tsc --outDir es --emitDeclarationOnly{{/if}}",
```

Comment:
can we put this in tsconfig, like does that work or its only an argument?

Reply:
yes, I think we can. for some reason I thought it made more sense to leave the tsconfig unopinionated on this matter 🤔

Reply:
generally i'd agree but we're kinda insisting on babel as the compiler for our setups so maybe it's a good idea?

Reply:
yep, I think you are right